dev-resources/template/2_lrs.yml

```diff
@@ -142,11 +147,14 @@ Resources:
     Type: AWS::Lambda::Function
     Properties:
       Code:
-        S3Bucket: !Ref S3Bucket
+        S3Bucket: !If
+          - SetDBInitFnBucket
+          - !Ref DBInitFnBucketOverride
+          - !Sub "yet-rds-db-init-deploy-${AWS::Region}"
```

Comment:
Do you know what would happen if you upload to a non-US region _without_ setting `DBInitFnBucketOverride` or `SetDBInitFnBucket`? Will it fail gracefully or will the whole thing explode?

Reply:
So you're asking what happens if you upload the DB function to a bucket in your region but don't set this? It will do what the template does currently, which is give a descriptive error saying you can't use a bucket in another region.

Reply:
Ok so it fails gracefully.

Reply:
Actually it is a bit different. It will say that it can't find a bucket _in your region_ with the name `yet-rds-db-init-deploy-<region name>`, but yeah as graceful as any other CloudFormation failure. Specification of more specific errors would be a nice CF feature but not one that exists.